langgraph_bot/nodes/triage_node.py

```
@@ -0,0 +1,110 @@
+            [
+                HumanMessage(
+                    content=TRIAGE_PROMPT.format(title=title, description=description)
+                )
+            ],
+            config={
+                "run_name": "triage",
+                "tags": [state.get("name") or "unknown-source"],
+                "metadata": {"raw_id": state.get("raw_id")},
+            },
+        )
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)